api/tests/integration/infrastructure/utils/csv/certification-results_test.js

```diff
@@ -57,6 +58,65 @@ describe('Integration | Infrastructure | Utils | csv | certification-results', (
     });
   });
 
+  describe('#getSessionCertificationResultsCsvWithCleaStatus', () => {
+
+    describe('when at least one certification has passed CleA certification', () => {
```

Comment:
Pas sûr de comprendre le describe, on veut un candidat ayant à la fois passé une certif Pix et une certif CléA Numérique ?

Reply:
Quand tu es positionné Cléa, lors de ta certif pix, tu passes aussi la certif cléa. Donc ici, on veut voir si un candidat à passé son cléa pendant sa certif Pix

Reply:
C'est ça ! C'est la double certification chez Pix. Ceux qui ont eux un badge Cléa numérique, lorsqu'ils passent leur certification Pix on les évalue aussi sur les compétences Cléa, et on leur donne une double certif s'ils ont réussit

Reply:
Ok c'est bien ce à quoi je m'attendais, en revanche je trouve la formulation peu claire. Je serais parti sur "When at least one certification was also a CléA Certification" ou "When at least one (student?) has passed certification and CléA certification". Vous voyez ce que je ne trouve pas clair. ?

Reply:
Si j'ai bien compris c'est le terme "certification" qui ne va pas (c'est vrai que c'est surement trop tech). On peut le remplacer par "candidate" qui correspondrait plus ? 
'when at least one candidate has passed CleA certification'